### ISSUE TITLE
jiti is processing .pnp.js

### ISSUE THREAD

**Issue by danielroe** [unjs/jiti#24]
Getting this error in a Yarn 2 project, sporadically:
```
 ERROR  [undefined] The whole PnP file got strict-mode-ified, which is known to break (Emscripten libraries aren't strict mode). This usually happens when the file goes through Babel.
at /my/repo

  at /my/repo
  at .pnp.js:7:9
  at v (.yarn/cache/jiti-npm-1.6.2-5ebc0543b7-7f4fd98426.zip/node_modules/jiti/dist/jiti.js:1:41499)
  at getPnpApi (.yarn/cache/typescript-patch-99fe32b02b-a8956044aa.zip/node_modules/typescript/lib/typescript.js:40348:12)
  at getPnpTypeRoots (.yarn/cache/typescript-patch-99fe32b02b-a8956044aa.zip/node_modules/typescript/lib/typescript.js:39236:18)
  at getDefaultTypeRoots (.yarn/cache/typescript-patch-99fe32b02b-a8956044aa.zip/node_modules/typescript/lib/typescript.js:39257:20)
  at getEffectiveTypeRoots (.yarn/cache/typescript-patch-99fe32b02b-a8956044aa.zip/node_modules/typescript/lib/typescript.js:39207:14)
  at Object.resolveTypeReferenceDirective (.yarn/cache/typescript-patch-99fe32b02b-a8956044aa.zip/node_modules/typescript/lib/typescript.js:39274:21)
  at loader_2 (.yarn/cache/typescript-patch-99fe32b02b-a8956044aa.zip/node_modules/typescript/lib/typescript.js:106672:90)
  at loadWithLocalCache (.yarn/cache/typescript-patch-99fe32b02b-a8956044aa.zip/node_modules/typescript/lib/typescript.js:106413:34)
  at actualResolveTypeReferenceDirectiveNamesWorker (.yarn/cache/typescript-patch-99fe32b02b-a8956044aa.zip/node_modules/typescript/lib/typescript.js:106673:140)
```

**Comment by pi0** [comment]
@danielroe Would you please create a minimal reproduction?

**Comment by danielroe** [comment]
Always :-)

**Comment by pi0** [comment]
@danielroe Or if you have a quick fix that would also work :)

**Comment by pi0** [comment]
Fixed in 1.6.4
